src/components/DropDown/DropDown.jsx

```diff
@@ -75,18 +77,19 @@ export default class DropDown extends Component {
   render () {
     const {
       children,
+      className,
       direction,
       pullRight,
       title
     } = this.props;
     const {isOpen} = this.state;
 
     return (
-      <div className={`${styles.container} ${styles[direction]} ${pullRight ? styles.pullRight : ''}`}>
+      <div className={`${styles.container} ${styles[direction]} ${pullRight ? styles.pullRight : ''} ${className}`}>
```

Comment:
We currently do not allow users to theme/style there own components via custom classNames.. I am not sure we should start doing this now 🤔

Reply:
I'll remove it, but I'll have to edit this component style on kununu-footer in a way or another :|